.github/scripts/addComponentLabel.js

```diff
@@ -2,27 +2,29 @@
 const { createLabelIfMissing } = require("./support/utils");
 
 /** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
-module.exports = async ({ github, context }) => {
+module.exports = async ({ github, context, core }) => {
   const { repo, owner } = context.repo;
+  const logParams = { title: "Add Component Label" };
 
   const payload = /** @type {import('@octokit/webhooks-types').IssuesEvent} */ (context.payload);
   const {
     issue: { body, number: issue_number },
   } = payload;
 
   if (!body) {
-    console.log("could not determine the issue body");
+    core.notice("could not determine the issue body", logParams);
```

Comment:
For consistency with other similar changes:
```suggestion
    core.notice("Could not determine the issue body", logParams);
```